### DISCUSSION TITLE
order of prediction box coordinates

### DISCUSSION THREAD

**Discussion by DSRajesh** [Project-MONAI/MONAI#7388] [Q&A]
Hello
I am interested in knowing some thing regarding the order of prediction box coordinates  in  "Lung nodule ct detection"  of MONAI ZOO.  How are the bounding box coordinates  presented  in the "result_luna16_fold0.json"    is it  --------------  xmin, xmax, ymin, ymax , zmin, zmax  ?    or      is it -------------------- x_center, y_center, z_center,  x_width, y_width, z_width   of the nodule ?

Thank you
Rajesh D S

**Comment by KumoLiu** [comment]
Hi @DSRajesh, you can refer to the comment here.
https://github.com/Project-MONAI/tutorials/discussions/1603#discussioncomment-8013229
Thanks.

**Reply by Rajesh-ParaxialTech** [reply]
Hello!!

![image](https://github.com/Project-MONAI/MONAI/assets/159233894/5e1b1e90-d10d-4632-a219-ce6d8a3fcdb9)


In the CCCWHD abbreviation above, W stands for width of the nodule, H for height and D for depth. And the units of dimension is mm ? Please let me know. Am i right ?

Thank you

Rajesh
